src/chariot/test/mongo-db.py

```diff
@@ -0,0 +1,5 @@
+from chariot.core.database.MongoDatabase import MongoDatabase
+
+mongo = MongoDatabase()
```

Comment:
Please add some structured tests if you want this file in master.

Reply:
This comment still stands. There is no use of pytest here and as such this is not a valid test file.